examples/federation-workshop/cluster-1/sentinel/Deployment.yaml

```diff
@@ -0,0 +1,76 @@
+# /*
+# |    Protect your secrets, protect your sensitive data.
+# :    Explore VMware Secrets Manager docs at https://vsecm.com/
+# </
+# <>/  keep your secrets… secret
+# >/
+# <>/' Copyright 2023–present VMware, Inc.
+# >/'  SPDX-License-Identifier: BSD-2-Clause
+# */
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vsecm-sentinel
+  namespace: vsecm-system
+  labels:
+    app.kubernetes.io/name: vsecm-sentinel
+    app.kubernetes.io/part-of: vsecm-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: vsecm-sentinel
+      app.kubernetes.io/part-of: vsecm-system
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: vsecm-sentinel
+        app.kubernetes.io/part-of: vsecm-system
+    spec:
+      serviceAccountName: vsecm-sentinel
+      containers:
+        - name: main
+          image: localhost:32000/vsecm-ist-sentinel:0.21.2
+          volumeMounts:
+            - name: spire-agent-socket
+              mountPath: /spire-agent-socket
+              readOnly: true
+          #
+          # You can configure VSecM Sentinel by providing environment variables.
+          #
+          # See https://vsecm.com/configuration for more information about
+          # these environment variables.
+          #
+          # When you don’t explicitly provide env vars here, VMware Secrets Manager
+          # Sentinel will assume the default values outlined in the given link above.
+          #
+          env:
+            - name: SPIFFE_ENDPOINT_SOCKET
```

Comment:
Glad to see the environment variables providing flexibility in connecting different clusters. 

As long as there’s network visibility and proper SPIRE federation, sky is the limit.